CMakeLists.txt

```diff
@@ -68,6 +65,12 @@ set(
   "Override C++ compiler optimization flags"
 )
 
+option(
+  IS_ZETH_PARENT
```

Comment:
This is to avoid collisions in target names

See, these files where this variable is set:
- https://github.com/scipr-lab/libff/blob/master/CMakeLists.txt#L83-L87
- https://github.com/scipr-lab/libfqfft/blob/master/CMakeLists.txt#L31-L35
- https://github.com/clearmatics/libsnark/blob/master/CMakeLists.txt#L84-L88

And see here where we set it to OFF: https://github.com/clearmatics/zeth/blob/master/depends/CMakeLists.txt#L2

Reply:
OK.  So this variable means "Zeth is the top-level project, and therefore it is ok to define targets which may conflict with other projects".  Might be worth a comment to clarify.

Reply:
I'll add a comment, thanks @dtebbs